src/components/__tests__/Header.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import Header from '../Header';
+
+describe('snapshot test for Header Component', () => {
+  test('renders correctly', () => {
+    const tree = renderer
+      .create(
+        <BrowserRouter>
+          <Header />
+        </BrowserRouter>,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
- [ ] This test is still failing, please fix this issue 😄 
    - Remember that every time you change something in the component you have to update the snapshot 🚀 
    ![imagen](https://user-images.githubusercontent.com/60789317/136432800-e5af5b7b-f6e6-4447-99de-22d2ec3233a1.png)